mpa/seg/semisl/inferrer.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from contextlib import nullcontext
+import os.path as osp
+
+import mmcv
+from mmcv.parallel import MMDataParallel
+from mmcv.runner import load_checkpoint, wrap_fp16_model
+from mmseg.datasets import build_dataloader, build_dataset
+from mmseg.models import build_segmentor
+from mpa.modules.hooks.recording_forward_hooks import FeatureVectorHook
+from mpa.registry import STAGES
+from .stage import SemiSegStage
+from mpa.stage import Stage
+import torch
+
+
+@STAGES.register_module()
+class SemiSegInferrer(SemiSegStage):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def configure(self, model_cfg, model_ckpt, data_cfg, training=False, **kwargs):
+        cfg = super().configure(model_cfg, model_ckpt, data_cfg, training=training, **kwargs)
+
+        cfg.model.type = cfg.model.orig_type
+        cfg.model.pop("orig_type", False)
+        cfg.model.pop("unsup_weight", False)
+        cfg.model.pop("warmup_start_iter", False)
```

Comment:
from `warmup_start_iter`to `semisl_start_iter`?

Reply:
You found a potential bug. Thanks. Revised.